engine/tests/mechanics/games/test_nim_game.py

```
@@ -19,64 +19,165 @@ class NimBlock(HasGame, Block):
+        )
+        handler._preselect_opponent_move(game)
+
+        # 7 -> take 3 -> 4, a losing position for whoever moves next
+        assert game.opponent_next_move == NimMove(pile=DEFAULT_HEAP, count=3)
+
+
+class TestMultipleHeaps:
+    """More than one heap, or more than one token type, is the same thing."""
+
+    def test_heaps_are_token_types_in_one_wallet(self) -> None:
+        game, handler = _ready(opening_heaps={"red": 5, "blue": 3})
```

Ruff (0.16.3)

[warning] Unpacked variable `handler` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)